lib/src/presentation/view/card_detail.dart

```diff
@@ -28,13 +28,21 @@ class _CardDetailState extends State<CardDetail> {
           iconTheme: IconThemeData(
             color: Constants.iconThemeColor,
           ),
-          title: Text(
-            cardInfo.name,
-            style: TextStyle(
-              color: Constants.iconThemeColor,
-              fontSize: Constants.titleFontSize,
-            ),
-          ),
+          title: cardInfo.name != null
+              ? Text(
+                  cardInfo.name!,
+                  style: TextStyle(
+                    color: Constants.iconThemeColor,
+                    fontSize: Constants.titleFontSize,
+                  ),
+                )
+              : Text(
+                  Constants.unknownCardName,
+                  style: TextStyle(
+                    color: Constants.iconThemeColor,
+                    fontSize: Constants.titleFontSize,
+                  ),
+                ),
```

Comment:
Here instead to check outside the Text constructor you can make the same but inside
Text(
                  cardInfo.name != null ? cardInfo.name! : Constants.unkownCardName,
                  style: TextStyle(
                    color: Constants.iconThemeColor,
                    fontSize: Constants.titleFontSize,
                  ),
                )

Reply:
Done